lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +209,40 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self._seed_urls = config.get_seed_urls()
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        all_links_bs = article_bs.find_all('a')
+        for link_bs in all_links_bs:
+            link = link_bs.get('href')
+            if link is None:
+                continue
+            elif link[0:8] == 'https://' and 'news' in link and 'from' in link:
+                self.urls.append(link)
+        return ''
```

Comment:
is there a more simple method of extracting link?
why you return '' here?

Reply:
I tried to return link, but then I could extract only one link from a seed url. So I decided to append links to self.url in a loop in order to extract all links from one seed url. And as I append them here, I don't need to return a link. I'm not sure about this return, though.

Reply:
Done